templates/Compensate_Image_Data.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    stitched_dir = os.path.join(folder_path, 'stitched_images')
</pre><p>might want to switch this lingo to 'tiled' instead of 'stitched'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>